### PR TITLE
Cumulus 1627

### PR DESCRIPTION
**Pull request by Jkovarik**
Updates CMA-js to utililze forthcoming CMA release with a pre-packaged CMA executable. 

This PR includes a version up as well.  This is a breaking change.

### CODE REVIEW

**Comment by Jkovarik** [comment]
These tests pass -locally- with the updated CMA.   This PR cannot be merged until CMA 1.1.2 is released.

File: index.js [hunk 1]
```diff
@@ -23,36 +24,27 @@ class CumulusMessageAdapterExecutionError extends Error {
...
-    });
-
-    // If STDIN is closed already, something went wrong.  Don't handle it
-    // here and expect a non-zero code when the close event fires.
-    // If this line is not present, calling sled.stdin.end with a chunk will
-    // result in an unhandled "Error: write EPIPE".
+async function callCumulusMessageAdapter(command, input) {
+  const adapterDir = process.env.CUMULUS_MESSAGE_ADAPTER_DIR || './cumulus-message-adapter';
+  const systemPython = await lookpath('python');
+  let spawnArguments = [systemPython, [`${adapterDir}`, command]];
```
Comment (yjpa7145) [inline index.js]: This feels a little easier to read:

```js
let spawnArguments;
if (systemPython) {
  spawnArguments = [systemPython, [`${adapterDir}`, command]];
} else {
  spawnArguments = [`${adapterDir}/cma`, [command]];
}
```

or

```js
const spawnArguments = systemPython
  ? [systemPython, [`${adapterDir}`, command]]
  : [`${adapterDir}/cma`, [command]];
```

File: index.js [hunk 2]
```diff
@@ -23,36 +24,27 @@ class CumulusMessageAdapterExecutionError extends Error {
...
+  // If there is no system python, attempt use of pre=packaged CMA binary
```
Comment (yjpa7145) [inline index.js]:
```suggestion
  // If there is no system python, attempt use of pre-packaged CMA binary
```

File: index.js (hunk 1 above)
Comment (Jkovarik) [reply index.js]: The default with catch block idiom tends to result in lower logic complexity/indentation, generally,  .... but fair enough, at the expense of explicit readability, particularly in simple cases.       

bbc5c29

File: index.js [hunk 3]
```diff
@@ -23,36 +24,32 @@ class CumulusMessageAdapterExecutionError extends Error {
...
-    cumulusMessageAdapter.on('close', (code) => {
-      if (code === 0) resolve(JSON.parse(cumulusMessageAdapterStdout));
-      else reject(new CumulusMessageAdapterExecutionError(cumulusMessageAdapterStderr));
-    });
+  let spawnArguments;
+  if (systemPython) {
+    spawnArguments = [systemPython, [`${adapterDir}`, command]];
+  }
+  else {
+    // If there is no system python, attempt use of pre=packaged CMA binary
```
Comment (yjpa7145) [inline index.js]:
```suggestion
    // If there is no system python, attempt use of pre-packaged CMA binary
```

**Pull request merged**